test/bigtest/interactors/settings/mappingProfilesInteractor.js

```diff
@@ -0,0 +1,15 @@
+import {
+  interactor,
+  scoped,
+} from '@bigtest/interactor';
+
+import SearchFormInteractor from '@folio/stripes-data-transfer-components/lib/SearchForm/tests/interactor';
```

Comment:
I think, ui-data-export shouldn't know so many details (folders structure) about stripes-data-transfer-components. If there there are any structure changes in stripes-data-transfer-components, we will have to update ui-data-export either. It seems, it doesn't sound good. Maybe, we should export Interactors for components, etc.

Reply:
I think this is applicable to tests only. We already do [this|https://github.com/folio-org/ui-data-export/blob/master/test/bigtest/interactors/queryFileUploaderInteractor.js#L10] for interactors from stripes-components and for CSS files. I think such changes should be done with extra precaution by checking affected parties.

Reply:
Have discussed with @YevheniiMaltsev and as a result, the [PR to stripes-data-transfer-components|https://github.com/folio-org/stripes-data-transfer-components/pull/24] has been created #31